packages/flutter/test/material/dropdown_test.dart

```diff
@@ -203,6 +203,47 @@ void main() {
     expect(value, equals('two'));
   });
 
+  testWidgets('Dropdown form field', (WidgetTester tester) async {
+    String value = 'one';
+
+    await tester.pumpWidget(
+      new StatefulBuilder(
+        builder: (BuildContext context, StateSetter setState) {
+          return new MaterialApp(
+            home: new Material(
+              child: new DropdownButtonFormField<String>(
+                value: value,
+                hint: const Text('Select Value'),
+                decoration: const InputDecoration(
+                  prefixIcon: const Icon(Icons.fastfood)
+                ),
+                items: menuItems.map((String val) {
+                  return new DropdownMenuItem<String>(
+                    value: val,
+                    child: new Text(val)
+                  );
+                }).toList(),
+                onChanged: (String v) {
+                  setState(() {
+                    value = v;
+                  });
+                },
+                validator: (String v) => v == null ? 'Must select value' : null,
+              ),
+            ),
+          );
+        }
+      )
+    );
+
+    await tester.tap(find.text('one'));
+    await tester.pump(const Duration(seconds: 1));
+    await tester.tap(find.text('three').last);
+    await tester.pump();
+    await tester.pump(const Duration(seconds: 1));
+    expect(value, equals('three'));
```

Comment:
add a control expectation before the awaits that checks that value equals 'one', just to make sure we don't "fix" this test one day by just changing the initial value